src/librustc_middle/query/mod.rs

```diff
@@ -214,6 +214,12 @@ rustc_queries! {
             cache_on_disk_if { key.is_local() }
         }
 
+        query coverage_data(key: DefId) -> Option<mir::CoverageData> {
+            desc { |tcx| "retrieving coverage data, if computed from MIR for `{}`", tcx.def_path_str(key) }
```

Comment:
nit: if this description is showing up in our backtrace, it's very likely that the "if computed" clause is true, so I'd suggest taking it out for clarity

Reply:
done